src/thirdparty/php.js

```diff
@@ -66,7 +66,15 @@ define(function(require, exports) {
   const format = (code, customConfiguration) => {
     // use custom configuration here. if there was none found, customConfiguration will still be null
     console.log('Using the following configuration', customConfiguration);
-
+      
```

Comment:
I already did the piece of code to read the preferences and handle the promises logic. The variable `customConfiguratuon` already loads the preferences. You could use that variable.

To test it, open a folder (project) in brackets that contains a `.phpbeautifyrc.json`, you will see the setting will be loaded and logged into the console by line 68